zipkin-server/src/main/resources/zipkin-server.yml

```diff
@@ -0,0 +1,152 @@
+zipkin:
```

Comment:
I wonder if we couldn't use yaml tricks to avoid this sort of copy/paste? ex include `zipkin-server.yml` in a file like `zipkin-server-aws.yml`? cc @shakuzen